quartz/plugins/transformers/ofm.ts

```diff
@@ -492,6 +494,17 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
                     "data-callout-metadata": calloutMetaData,
                   },
                 }
+
+                // Add callout-content class to callout body if it has one.
+                if (calloutContent !== undefined) {
+                  const calloutContentWrapped: BlockContent | DefinitionContent = {
+                    type: "html",
+                    value: `<div class="callout-content">
+                      ${toHtml(toHast(calloutContent, { allowDangerousHtml: true }), { allowDangerousHtml: true })}
```

Comment:
@jackyzha0 I noticed there is a `mdastToHtml` function in `ofm.ts`, but its typing doesn't allow usage here. Is it acceptable to expand the typing? Or should I look into a different solution?

Reply:
urgh i don't like this rendering approach either, let's just go back to blockquote with hName